assets/constants.js

```diff
@@ -607,5 +607,7 @@ var constants = {
   // eslint-disable-next-line max-len
   "DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR": "Check out this interactive lesson I created on Oppia - a free platform for teaching and learning!",
 
+  "GCS_RESOURCE_BUCKET_NAME": "None-resources",
```

Comment:
Why `None-resources`?

Reply:
The resource bucket name isn't used in dev mode, but on prod mode that hasn't been deployed, it has to be a non-null value. So I ran it on prod mode to confirm the value and it was `None-resources`. So I used that.